packages/node_modules/@node-red/editor-api/lib/editor/index.js

```diff
@@ -116,6 +116,11 @@ module.exports = {
             // SSH keys
             editorApp.use("/settings/user/keys",needsPermission("settings.write"),info.sshkeys());
 
+            // module
+            var module = require("./module");
+            module.init(runtimeAPI);
+            editorApp.post("/module/create/subflow", needsPermission("module.write"), module.createSubflow);
```

Comment:
We need to discuss the choice of URL here - it wasn't mentioned in the design document.

We do not currently have anything under `/module` - and I'm not aware of anything else planned that would fit under it. I am not sure if that is the right top-level path to use here.

We don't have any other end points that include the verb (`create`) in them.

We currently have top-level end points of:
 - `/nodes`
 - `/flows`
 -  `/flow`

In the future we might have a `/subflow` end point that will make it easier to add/remove subflows (similar to the `/flow` end point).

With that in mind, I think the endpoint should be `POST /subflow/module`.

Reply:
I assumed that other kind of NPM modules may be downloaded via this endpoint. 
But, in new commit, changed it according to your suggestion.